src/components/Banner/Banner.tsx

```diff
@@ -106,7 +106,7 @@ const variantToIconAssetsMap: {
  * <Banner
  *   onDismiss={handleDismiss}
  *   title="Some Title"
- *   description={<>Some description, possibly with a <Link href="www.some-other-resource.com">link to some other resource</Link>.</>}
+ *   description={<>Some description, possibly with a <Link href="https://go.czi.team/eds">link to some other resource</Link>.</>}
```

Comment:
Made this change because this is the `href` we used in the button stories on `main`.